From 9dcdd062c899bc6f1acfbf39e0d3d1cc3237d8fa Mon Sep 17 00:00:00 2001
From: kboeluek <kboeluek@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 15 May 2014 18:34:37 +0000
Subject: [PATCH 0467/1000] MANTIS 0020990 change; Platform_Extension_CUSS_1_2:
 Periodic/Automatic restart of the application

- changed types of data members from string to unsigned long

git-svn-id: svn://localhost/SelfServiceCommon/trunk@676 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/appcontrol.idl | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/appcontrol.idl b/SelfServiceCommon/Massai/idl/Massai/appcontrol.idl
index 2ead3e65..ed67ab8c 100644
--- a/SelfServiceCommon/Massai/idl/Massai/appcontrol.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/appcontrol.idl
@@ -22,7 +22,7 @@
 
 module appControl
 {
-    const string interface_version = "1.1";
+    const string interface_version = "1.2";
    
     // list of strings
     typedef sequence<string> strings;
@@ -95,8 +95,8 @@ module appControl
     struct data
     {
       string applicationName;
-      string applicationMemoryUsage;
-      string applicationTimeUsage;
+      unsigned long applicationMemoryUsage;
+      unsigned long applicationTimeUsage;
     };
 };
 
-- 
2.41.0.windows.1

